utils/k8s/k8s.go

```diff
@@ -310,22 +320,52 @@ func (k8s K8S) ExecToPodThroughAPI(command, podName, namespace string, stdin io.
 	return stdout.String(), stderr.String(), nil
 }
 
+// ExecToPodThroughKubectl uninterractively exec to the pod with the command specified using `kubectl exec`
```

Comment:
same here !!

Reply:
One more comment @a4abhishek , Please correct the word trhough to through in the commit message !!